perun-cli/Perun/AttributesAgent.pm

```diff
@@ -170,7 +180,17 @@ sub checkAttributeValue
 #(resource => $resourceId, member => $memberId, attributes => $arrayOfAttributes)
 sub checkAttributesValue
 {
-	return Perun::Common::callManagerMethod('checkAttributeValue', '', @_);
+    print STDERR "Warning: method 'checkAttributeValue' is deprecated. Please use 'checkAttributeSemantics' and 'checkAttributeSyntax' instead.";
+	return Perun::Common::callManagerMethod('checkAttributeSemantics', '', @_);
```

Comment:
Please change callManagerMethod to "checkAttributesSemantics" instead of checkAttributeSemantics.